### PR TITLE
[ISSUE #465] Add friendly reminder for the lack of cluster store topic

### LINKED ISSUE #465: The behavior of the cluster store topic leads to confusion
When broker's setting item autoCreateTopicEnable is true, configStoreTopic、positionStoreTopic、offsetStoreTopic will be created automatically. When autoCreateTopicEnable is false, there will be a topic not exist exception.
But clusterStoreTopic won't be created and no warning whatever autoCreateTopicEnable is.

### PR DESCRIPTION
**Pull request by ShadowySpirits**
close #465

### CODE REVIEW

File: rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java [hunk 1]
```diff
@@ -81,6 +81,14 @@ public void stop() {
         this.defaultMQPullConsumer.shutdown();
     }
 
+    @Override
+    public boolean hasClusterStoreTopic() {
```
Comment (duhenglucky) [inline rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java]: maybe there's a config  to decide whether to automatically create a topic.

Comment (ShadowySpirits) [reply rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java]: Wether automatically create a topic or not depends on the config of broker. I didn't find a mechanism to manually create a topic from the client. Therefore add autoCreateTopicEnable to connector maybe lead to misunderstanding.

Comment (duhenglucky) [reply rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java]: sorry, I thought you wanted to create a topic in here.

**Comment by duhenglucky** [comment]
@ShadowySpirits merge this PR first, but it would be nice if you can check all the necessary resources during the startup phase, instead of checking them in Rebalance service.

**Pull request merged**